### PR TITLE
Update man pages for schema commands

### PR DESCRIPTION
**Pull request by KevinAnsleyJohnson**
Signed-off-by: Kevin Johnson <kevin_johnson@cargill.com>

### CODE REVIEW

**Review by vaporos — CHANGES_REQUESTED** [review]
See comments on #736, which would apply here as well.

**Review by vaporos — APPROVED** [review]

File: cli/man/grid-schema-create.1.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+`-v`
+: Increases verbosity (the opposite of `-q`). Specify multiple times for more
+  output.
+
+OPTIONS
+=======
+
+`-k`, `--key`
+: base name or path to a private signing key file.
```
Comment (dplumb94) [inline cli/man/grid-schema-create.1.md:49]: b should be capitalized

**Review by dplumb94 — CHANGES_REQUESTED** [review]

**Review by dplumb94 — APPROVED** [review]

File: cli/man/grid-schema-create.1.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+`--url`
+: URL for the REST API.
+
+`--wait`
+: Maximum number of seconds to wait for the batch to be committed.
+
+EXAMPLES
+========
+
+schemas can be created by using the `create` command.
```
Comment (shannynalayna) [inline cli/man/grid-schema-create.1.md:64]: 'schemas' should be capitalized here

File: cli/man/grid-schema-update.1.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+`--url`
+: URL for the REST API.
+
+`--wait`
+: Maximum number of seconds to wait for the batch to be committed.
+
+EXAMPLES
+========
+
+schemas can be updated by using the `update` command.
```
Comment (shannynalayna) [inline cli/man/grid-schema-update.1.md:64]: 'schemas' should be capitalized here as well

**Review by shannynalayna — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]

**Pull request merged**